docs/2.9.3/docs/tools/daml-shell/index.rst

```diff
@@ -50,11 +50,11 @@ Running Daml Shell
 ==================
 
 You can run Daml Shell from a ``jar`` artifact or from a ``Docker``
-image.  See the Release Notes for download instructions.
+image. See the Release Notes for download instructions.
 
 .. code-block:: sh
 
-   docker run -it --net host docker.io/digitalasset/daml-shell:<version-tag> # change --net host if desired
+   docker run -it --net host digitalasset-docker.jfrog.io/daml-shell:<version-tag> # change --net host if desired
```

Comment:
An example URL for a specific tag would be useful - it's not obvious where the user should type daml-shell:v0.1.4 or daml-shell:0.1.4 here.